File: payroll-api/src/main/java/com/company/payroll/model/Department.java
package com.company.payroll.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@RequiredArgsConstructor
public class Department {
//    private Integer deptno;
//
//    private String deptname;
//
//    private String location;
//
//    private String state;
//
//    private String country;

    private Integer deptNo;
    private String deptName;
    private String deptAddress;
    private Integer postalCode;
    private String state;
    private String country;
}

File: kotlin-payroll/src/main/kotlin/com/company/payroll/model/Department.kt
package com.company.payroll.model

import com.company.payroll.annotation.NoArg

@NoArg
data class Department(
    var deptNo: Int,
    var deptName: String,
    var deptAddress: String,
    var postalCode: Int,
    var state: String,
    var country: String
)
